### ISSUE TITLE
Hyperwave Comm Scanner not working

### ISSUE THREAD

**Issue by echoesandechoes** [xws-bench/bench#40]
Hi,
Hyperwave Comm Scanner is not working properly, at least when added to the Upsilon Shuttle. The match won't proceed to Turn 1.
Thanks

**Comment by xws-bench** [comment]
Yes, indeed. It's a bit less buggy (does not stall the game) but does not correspond so far to what is written on the card. Timing issue.
